alibi_detect/cd/context_aware.py

```diff
@@ -88,7 +100,7 @@ def __init__(
 
         kwargs = locals()
         args = [kwargs['x_ref'], kwargs['c_ref']]
-        pop_kwargs = ['self', 'x_ref', 'c_ref', 'backend', '__class__']
+        pop_kwargs = ['self', 'x_ref', 'c_ref', 'backend', '__class__', 'inputs']
```

Comment:
Why do we ned `inputs` here?

Reply:
It was there from where we had `inputs = locals().copy()` but you're right it should be removed now.